scopes/explorer/ui/base-component-card/details/details.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import styles from './details.module.scss';
+
+export type ComponentDetailsProps = {
+  id: string;
+  version?: string;
+  description?: string;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+export function ComponentDetails({ id, version, description, className, ...rest }: ComponentDetailsProps) {
+  const idArray = id.split('/');
```

Comment:
at least add comment to use component id

Reply:
done